terminal.py

```diff
@@ -19,7 +19,7 @@ def main():
     else:
         from openbb_terminal.core.session import session_controller
 
-        if is_auth_enabled():
+        if is_auth_enabled() and ("--login" in sys.argv[1:] or is_installer()):
```

Comment:
Will installer users need to set the flag in the `.env` to test the login functionality?

Reply:
i just make this function return True lol.  But until we launch, yeah